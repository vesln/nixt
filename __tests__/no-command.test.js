const nixt = require('..');

describe('nixt#run', () => {
  it('throws an error when no command is supplied', () => {
    expect(() => {
      nixt().end(() => {});
    }).toThrowErrorMatchingSnapshot();
  });
});
