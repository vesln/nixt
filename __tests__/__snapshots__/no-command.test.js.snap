// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`nixt#run throws an error when no command is supplied 1`] = `"Please provide a command to run. Hint: \`nixt#run\`"`;
